### PR TITLE
Add PPTL() command

### PR DESCRIPTION
**Pull request by acarbonetto**
*Issue #, if available:*

*Description of changes:*

Adds the [PTTL](https://redis.io/commands/pttl/) command to the java wrapper

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: java/client/src/main/java/glide/api/commands/GenericBaseCommands.java [hunk 1]
```diff
@@ -262,11 +262,29 @@ CompletableFuture<Boolean> pexpireAt(
...
+     * assert timeRemaining == -2L; // Returns -2 for a non-existing key.
      * }</pre>
      */
     CompletableFuture<Long> ttl(String key);
+
+    /**
+     * Returns the remaining time to live of <code>key</code> that has a timeout, in milliseconds.
+     *
+     * @see <a href="https://redis.io/commands/pttl/">redis.io</a> for details.
+     * @param key - The key to return its timeout.
```
Comment (SanHalacogluImproving) [inline java/client/src/main/java/glide/api/commands/GenericBaseCommands.java:277]: Remove -

File: java/client/src/main/java/glide/api/commands/GenericBaseCommands.java [hunk 2]
```diff
@@ -262,11 +262,29 @@ CompletableFuture<Boolean> pexpireAt(
...
+     * @return TTL in milliseconds. <code>-2</code> if <code>key</code> does not exist, <code>-1
```
Comment (SanHalacogluImproving) [inline java/client/src/main/java/glide/api/commands/GenericBaseCommands.java:278]: Line breaks?

File: java/client/src/main/java/glide/api/models/BaseTransaction.java [hunk 3]
```diff
@@ -1241,6 +1242,21 @@ public T zcard(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Returns the remaining time to live of <code>key</code> that has a timeout, in milliseconds.
+     *
+     * @see <a href="https://redis.io/commands/pttl/">redis.io</a> for details.
+     * @param key - The key to return its timeout.
```
Comment (SanHalacogluImproving) [inline java/client/src/main/java/glide/api/models/BaseTransaction.java:1249]: Remove -

**Review by SanHalacogluImproving — APPROVED** [review]

File: java/client/src/main/java/glide/api/commands/GenericBaseCommands.java (hunk 2 above)
Comment (acarbonetto) [reply java/client/src/main/java/glide/api/commands/GenericBaseCommands.java:278]: Nah!  I think this is fine as a single paragraph.

**Pull request merged**